app/src/main/java/fr/free/nrw/commons/profile/achievements/AchievementsFragment.java

```diff
@@ -179,11 +181,38 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
             tvAchievementsOfUser.setVisibility(View.VISIBLE);
             tvAchievementsOfUser.setText(getString(R.string.achievements_of_user,userName));
         }
+
+        // Achievements currently unimplemented in Beta flavor. Skip all API calls.
+        if(ConfigUtils.isBetaFlavour()) {
+            progressBar.setVisibility(View.GONE);
+            imageByWikiText.setText(R.string.no_image);
+            imageRevertedText.setText(R.string.no_image_reverted);
+            imageUploadedText.setText(R.string.no_image_uploaded);
+            wikidataEditsText.setText("0");
+            imagesFeatured.setText("0");
+            tvQualityImages.setText("0");
+            thanksReceived.setText("0");
+            setMenuVisibility(true);
+            return rootView;
+        }
         setWikidataEditCount();
         setAchievements();
         return rootView;
     }
 
+    @Override
+    public void setMenuVisibility(boolean visible) {
```

Comment:
Add test when false is passed as the argument, in that case toast should not appear